disperser/dataapi/docs/swagger.yaml

```diff
@@ -258,6 +271,35 @@ paths:
       summary: Fetch non signers
       tags:
       - Metrics
+  /metrics/operator_nonsigning_percentage:
+    get:
+      parameters:
+      - description: 'Interval to query for non signers in seconds [default: 3600]'
```

Comment:
nit: this seems to be from the /metrics/non_signers. We may revise it to fit this API better, like "Interval to query for operator nonsigning percentage"